docs/source/user_guide/computed_properties.rst

```diff
@@ -510,23 +510,132 @@ Data Files in `Analysis/DOS/data`
+
+- :math:`T`: Temperature, fixed to 300 K,
+- :math:`M`: Mean number of nearest neighbors, fixed to 8.0,
+- :math:`C`: Empirical parameter, fixed to 0.25 to provide the best agreement with experiments for small organic molecules.
+- :math:`\lambda`: Reorganization energy, fixed to 200 meV.
+
+Computed Parameters
+~~~~~~~~~~~~~~~~~~~
+
+Parameters below are highly materials-specific and are computed with `Nanoscope`:
+
+- :math:`\sigma`: Energy disorder due to environmental variations in molecular energy levels, [:math:`eV`],
+- :math:`J^2 r^2`: Reflects the ease of charge transfer between molecules,  [:math:`eV²·Å²`].
+
+**Computed Mobility Data**
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The computed mobility for both holes and electrons is saved as a YAML file, which includes the parameter values used in the computation for reference.
+
+
+Output Files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The output of the GEMM-based mobility calculation is stored in the following file:
+
+::
+
+    ESAnalysis
+       └─── Analysis
+              └─── DOS
+                    └─── data
+                           └─── mobility.yaml
+
+File Content
+~~~~~~~~~~~~~
+
+.. raw:: html
+
+   <table class="docutils" style="width: 100%; table-layout: fixed; border-collapse: collapse;">
+      <thead>
+         <tr>
+            <th style="width: 20%; padding: 8px; border: 1px solid #ddd; text-align: left; overflow-wrap: break-word; white-space: normal;">File</th>
+            <th style="width: 80%; padding: 8px; border: 1px solid #ddd; text-align: left; overflow-wrap: break-word; white-space: normal;">Description</th>
+         </tr>
+      </thead>
+      <tbody>
+         <tr>
+            <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">mobility.yaml</td>
+            <td style="padding: 8px; border: 1px solid #ddd; overflow-wrap: break-word; white-space: normal;">Contains the computed values of hole and electron mobilities, along with detailed parameters used for the calculation.</td>
+         </tr>
+      </tbody>
+    </table>
+
+**Example:**
+~~~~~~~~~~~~
+
+.. code-block:: yaml
+
+    hole_mobility:
+      unit: cm²/V·s
+      value: 2.09e-5  # hole mobility in cm2/Vs
+      parameters:
+        J2_r2: 1.42e-3  # (eV²·Å²)
+        lambda_eV: 0.266  # eV
+        sigma: 0.13  # eV
+        T: 290  # K
+        M: 8.16
+        n: 3
+        C: 0.25
+    electron_mobility:
+      unit: cm²/V·s
+      value: 1.01e-10  # electron mobility in cm2/Vs
+      parameters:
+        J2_r2: 9.99e-3  # (eV²·Å²)
+        lambda_eV: 0.296
+        sigma: 0.224
+        T: 290
+        M: 7.31
+        n: 3
+        C: 0.25
+
 
-.. Content about Mobility.
 
 
 .. _overlap_integrals:
 
 .. Overlap Integrals
 .. -----------------
 
-.. Content about Overlap Integrals.
+Content about Overlap Integrals.
```

Comment:
These I commented for the purpose that we do not have empty headers, until we find time to fill